server/tests/test_inference.py

```
@@ -0,0 +1,67 @@
+            for token in runner.generate_streaming(prompt, max_tokens=100, json_schema=json.dumps(json_schema)):
+                print(token, end="", flush=True)
+                full_response += token
+            print("\n--- JSON Done ---\n")
+            
+            try:
+                parsed = json.loads(full_response)
+                print(f"Parsed JSON: {parsed}")
+                assert "answer" in parsed
+                assert "confidence" in parsed
+                print("JSON Schema verification PASSED")
+            except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)